python/ray/util/tracing/tracing_helper.py

```diff
@@ -84,11 +84,22 @@ def _try_import(self, module):
                 )
 
 
-_opentelemetry = _OpenTelemetryProxy()
-_opentelemetry.try_all()
-
 _nameable = Union[str, Callable[..., Any]]
 _global_is_tracing_enabled = False
+_opentelemetry = None
+
+
+def _is_tracing_enabled() -> bool:
+    """Checks environment variable feature flag to see if tracing is turned on.
+    Tracing is off by default."""
+    return _global_is_tracing_enabled
+
+
+def _enbale_tracing():
+    global _global_is_tracing_enabled
+    _global_is_tracing_enabled = True
+    _opentelemetry = _OpenTelemetryProxy()
```

Comment:
for my knowledge what does this do? does all our worker telemetry flow through this?

Reply:
it's an experimental that we merged long time ago for distributed tracing. I believe almost no one uses this feature now, but it has really tight integration to core